### DISCUSSION TITLE
How about updating "undo" so that it can be used multiple times?

### DISCUSSION THREAD

**Discussion by leemj4347** [ImageMagick/ImageMagick#6938] [Contribute]
### Describe the solution you'd like

How about updating "undo" so that it can be used multiple times? I think this can make it more convenient for users.

**Comment by urban-warrior** [comment]
We need context.  Do you mean Undo for the `display` utility?  We drew a black rectangle then a block circle.  We pressed Undo and the circle was removed.  We pressed it again and the rectangle was removed.   Provide steps that we can reproduce where Undo is not effective.

**Comment by fmw42** [comment]
Perhaps he means in the GUI or the web version

**Comment by leemj4347** [comment]
If I edit the transform such as flip or flop in "imagemagick display" and press undo, after undo, undo will be disabled and i will not be able to press it anymore. Also, if i change the size of the view several times and then run undo, it will only run once.
![스크린샷 2023-12-11 113644](https://github.com/ImageMagick/ImageMagick/assets/57532151/6a7d473c-cabc-47e5-9dfe-78f97eb1137b)

**Comment by urban-warrior** [comment]
The Windows IMDisplay app was developed by external contributors, and our team does not actively update or modify it. As ImageMagick is an open-source project, any community member has the opportunity to improve the app by submitting a git pull request.
